cluster/gce/gci/configure-helper.sh

```diff
@@ -1392,8 +1394,11 @@ function prepare-etcd-manifest {
     etcd_protocol="https"
   fi
 
-  if [[ -n "${ETCD_APISERVER_CA_KEY:-}" && -n "${ETCD_APISERVER_CA_CERT:-}" && -n "${ETCD_APISERVER_SERVER_KEY:-}" && -n "${ETCD_APISERVER_SERVER_CERT:-}" ]]; then
+  if [[ -z "$1" && -n "${ETCD_APISERVER_CA_KEY:-}" && -n "${ETCD_APISERVER_CA_CERT:-}" && -n "${ETCD_APISERVER_SERVER_KEY:-}" && -n "${ETCD_APISERVER_SERVER_CERT:-}" && -n "${ETCD_APISERVER_CLIENT_KEY:-}" && -n "${ETCD_APISERVER_CLIENT_CERT:-}" ]]; then
```

Comment:
Why do you care that the suffix is empty?

nit: use a named local variable in place of `$1`

Reply:
We only want mTLS enabled for etcd server but not etcd-events. if $1 suffix is empty, it's etcd server, otherwise it's etcd events server. Good reminder, I should add comment here and I added local variable suffix in place of $1.